doc/source/f2py/getting-started.rst

```diff
@@ -45,9 +45,9 @@ to run
 
 ::
 
-  f2py -c fib1.f -m fib1
+  python -mnumpy.f2py -c fib1.f -m fib1
```

Comment:
I think a space after `-m` would be usual way to write this. See `python -m pytest [...]` as an example.

Reply:
fixed